config/config_test.go

```diff
@@ -78,6 +78,54 @@ func TestOnlyEnvConfig(t *testing.T) {
 	os.Setenv("DD_API_KEY", "")
 }
 
+func TestOnlyEnvConfigArgsScrubbingEnabled(t *testing.T) {
+	os.Setenv("DD_SCRUB_ARGS", "true")
+	os.Setenv("DD_CUSTOM_SENSITIVE_WORDS", "*password*,consul_token,*api_key")
+
+	agentConfig, _ := NewAgentConfig(nil, nil)
+	assert.Equal(t, true, agentConfig.Scrubber.Enabled)
+
+	cases := []struct {
+		cmdline       []string
+		parsedCmdline []string
+	}{
+		{[]string{"spidly", "--mypasswords=123,456", "consul_token", "1234", "--dd_api_key=1234"},
+			[]string{"spidly", "--mypasswords=********", "consul_token", "********", "--dd_api_key=********"}},
+	}
```

Comment:
nit: the formatting is a bit weird here. maybe line break around the outside open/close brackets?